e2e/app/page/workspaces-page.ts

```diff
@@ -157,18 +156,6 @@ export default class WorkspacesPage extends WorkspaceEditPage {
     return notebookPage.goAnalysisPage();
   }
 
-  /**
-   * Click DELETE WORKSPACE button in delete workspace Confirmation dialog.
-   * @return {string} Dialog textContent.
-   */
-  async deleteWorkspaceModal(): Promise<string[]> {
```

Comment:
`deleteWorkspaceModal` moved to `workspace-base` and renamed to `dismissDeleteWorkspaceModal`.